theme/snippets/product-info-bar.liquid

```diff
@@ -1,9 +1,45 @@
-{% comment %} TO-DO: Adjust or replace Tags, Add "Add to Cart" functionality {% endcomment %}
+<!-- Check if product already exists in cart -->
+{% for item in cart.items %}
+  {% if item.product.id == product.id %}
+    {% assign product_in_cart = true %}
+    {% break %}
+  {% else %}
+    {% assign product_in_cart = false %}
+  {% endif %}
+{% endfor %}
+
+<!-- Get product type -->
+{% assign product_type = "" %}
+{% if product.first_available_variant.metafields.bookings != blank %}
+  {% assign product_type = product.first_available_variant.metafields.bookings.bookable_type %}
+{% else %}
+  {% assign product_type = product.type %}
+{% endif %}
+
```

Comment:
If it's a product created by Shopify Bookings, there will always be a `product.first_available_variant.metafields.bookings` and `product.first_available_variant.metafields.bookings.bookable_type`?

If Shopify Bookings product, `product_type` comes from Bookings, otherwise `product_type` comes from the Shopify product?

Reply:
Yeah exactly, 

Shopify bookings always creates a variant for any product it creates. Because we only have a single variant for each product (for now) we can determine the product_type by reading the first variants `bookable_type` attribute.

In the case that a product is added directly in Shopify (e.g. a book) we can tag it ourselves, and read that to determine the product type. 

So basically, for the time being we can skip adding product types to products generated by bookings, and just read from the bookable_type.

Reply:
Niiiice thank you!